release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1782,6 +1784,8 @@ FATAL failed to fetch Metadata: failed to load asset "Install Config": platform.
 
 *RHCOS*
 
+* Previously, multipath was enabled too late in the boot process. Consequently, {op-system-first} would hit I/O errors in some multipath environments. With this update, multipath is now enabled earlier in the boot process. As a result, RHCOS no longer hits I/O errors in some multipath environments.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954025[*BZ#1954025*])
```

Comment:
Just a nit observation:

- "...would hit I/O errors..." hit seems like the wrong word here? Would using the word, "return" change the meaning of the sentence? Same issue with the last sentence.